page/settings.py

```diff
@@ -35,15 +33,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'tasks',
-    'django.contrib.staticfiles',
-    'south',
-    'django.contrib.webdesign',
-    'submit',
-    'ksp_login',
-    'social.apps.django_app.default',
-    'about',
     'django_extensions',
+    'social_django',
+    'ksp_login',
+
+    'about.apps.AboutConfig',
```

Comment:
oki, ma to aj nejake vyhodi okrem toho, ze nemusis vypisovat celu cestu ku konfigu?